packages/dashboard/src/features/database/hooks/useRecords.ts

```
@@ -1,7 +1,10 @@
 import { ConvertedValue } from '#components/datagrid/datagridTypes';
 import { DEFAULT_DATABASE_SCHEMA } from '#features/database/helpers';
 import { databaseTableQueryKeys } from '#features/database/queryKeys';
-import { recordService } from '#features/database/services/record.service';
+import {
+  recordService,
+  type RecordPrimaryKey,
+} from '#features/database/services/record.service';
```

ESLint

[error] Replace `⏎··recordService,⏎··type·RecordPrimaryKey,⏎` with `·recordService,·type·RecordPrimaryKey·`

(prettier/prettier)